tests/console/console.spec.ts

```diff
@@ -0,0 +1,45 @@
+import Server from '../../src/server/server'
+
+jest.mock('net', () => {
+  type MaybeNumber = number | undefined
+  type MaybeString = string | undefined
+  type MaybeVoidFunction = (() => void) | undefined
+
+  return {
+    createServer: () => (
+      {
+        listen: (
+          port: MaybeNumber,
+          host: MaybeString,
+          callback: MaybeVoidFunction
+        ): void => {
+          if (callback === undefined) {
+            return
+          }
+          callback()
+        }
+      }
+    )
+  }
+})
+
+describe('Console', () => {
```

Comment:
This should be changed to 'Server' since we're testing the Server behavior, not the console behavior

Reply:
Also, the filename should change. The console is something created by JavaScript. It's not code we control. It outputting to the user as a consequence of the connection opening is what's being tested; _not_ the console itself.

I suggest putting this in a `server` folder instead of a `console` folder and renaming the file to `server.spec.ts`

Reply:
The directory, file, and test name were renamed to 'Server'. :)